### PR TITLE
chore: Spelling - md files

### PR DESCRIPTION
**Pull request by jsoref**
Split from #3647
Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Optional. My organization is added to USERS.md.
* [x] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: util/kustomize/testdata/kustomization_yaml/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```
Comment (igaskin) [inline util/kustomize/testdata/kustomization_yaml/statefulset.yaml:1]: why was this file added?

**Review by igaskin — COMMENTED** [review]
can you squash these commits?

File: util/kustomize/testdata/kustomization_yaml/statefulset.yaml (hunk 1 above)
Comment (jsoref) [reply util/kustomize/testdata/kustomization_yaml/statefulset.yaml:1]: Sigh. Bad split. It's from the other side.

Comment (jsoref) [reply util/kustomize/testdata/kustomization_yaml/statefulset.yaml:1]: Specifically, that commit was on both sides of this work...

**Review by alexmt — APPROVED** [review]
LGTM, thank you!
